plugins/neat.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: neat
+spec:
+  caveats: |
+    This plugin depends on the following binaries -
+    `jq`, `yq` and `sponge` (from `moreutils`), which should be installed by the user
```

Comment:
Here, I'd also recommend to put the tools into one line each, in order to increase visibility. For example:

```yaml
caveats: |
  This plugin depends on the following tool which need to be installed independently:
    * jq
    * yq
    * sponge (from moreutils)
```

And you could mention where to get help, like `kubectl neat -h`?

Reply:
I've broken the lines as you suggested.
My plugin doesn't have a built in help option, because it's by default wrapping kubectl get (so -h will show the get help currently). In the rewrite I will refactor the get wrapping into a get subcommand (kubectl neat get) and will also add help.